app/javascript/onboarding/components/ProfileForm/TextArea.jsx

```diff
@@ -0,0 +1,36 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import { FormField } from '@crayons';
+
+const TextArea = (props) => {
```

Comment:
nitpick (non-blocking) Consider using a function expression instead of an [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) in the new components created. Arrow functions are only required when you need access to the lexical scope, i.e. dealing with `this`.

Reply:
✅